src/architecture/configuration.md

```diff
@@ -92,17 +109,42 @@ The configuration used to create a genesis block.
 - **tcp_reconnect_timeout_max**  
   Maximum timeout (ms) for reconnect attempt.
 
-#### [node.api]
+#### [api]
 
 API configuration parameters.
 
 - **enable_blockchain_explorer**  
   Enable api endpoints for the blockchain explorer on the public API address.
+- **state_update_timeout**  
+  Timeout (ms) to update API state.
 - **public_api_address**  
   Listen address for public API endpoints.
 - **private_api_address**  
   Listen address for private API endpoints.
 
+#### [whitelist]
+
+Whitelist parameters.
+
+- **whitelist_enabled**  
+  Enable whitelisting.
+- **whitelisted_peers**  
+  List containing consensus public keys of trusted peers.
+
+#### [mempool]
+
+Configuring queues sizes.
+
+- **events_pool_capacity**  
+  Maximum number of events in the [event queue](../advanced/consensus/specification.md#message-processing).
+
+- **tx_pool_capacity**  
+  Maximum number of transactions in the pool of unconfirmed transactions.
+
+#### [services_configs]
+
+Service-specific parameters.
```

Comment:
Please specify the format and/or link to [the Services article](https://github.com/exonum/exonum-doc/blob/master/src/architecture/services.md#service-identifiers). Also, the text there says

> Service configuration is stored in the overall configuration under the key `service_name` in the `services` variable

while here the variable is `services_configs` (which btw seems wrong from the language perspective; `service_configs` would be correct). Which one is true? If `services_configs`, then please edit the Services article within this PR, because the update would concern configuration.